gce/survival-training/README.md

```diff
@@ -0,0 +1,113 @@
+# Compute Engine survival training
+
+Training jobs that survive Compute Engine shutdowns
+
+- - -
+
+[Google Compute Engine](https://cloud.google.com/compute/docs/) (Compute Engine) gives you direct access to the computational backbone of Google Cloud Platform. You can use Compute Engine to create virtual machines provisioned with the resources of your choosing ([satisfying the constraints of any quotas that you define](https://cloud.google.com/compute/quotas)).
+
+This has particular value when it comes to training machine learning models, as this gives you access to resources beyond those available to you physically. Now Compute Engine offers preemptible GPU instances. This means that, if you design your trainer appropriately, your cost per training step could be a fraction of what it would be on a dedicated instance.
+
+Dedicated GPU-enabled Compute Engine instances go down regularly for maintenance and, by default, restart but without reinitializing the processes that were running on them before the shutdown. This introduces significant overhead in both time and cost to performing training on Compute Engine instances.
```

Comment:
> Dedicated GPU-enabled Compute Engine instances go down regularly for maintenance and, by default, restart but without reinitializing the processes that were running on them before the shutdown.

Dedicated GPU-enabled Compute Engine instances go down regularly for maintenance, and they restart without reinitializing the processes that were running on them before the shutdown.

Reply:
Changed, merged with subsequent paragraph, too.